Packs/AWS-ACM/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### AWS - ACM
+- Internal code improvements.
```

Comment:
can rm. not needed when we have detailed rn. any pr contains also some internal code improvements..